docs/src/pages/system/basics/Demo.tsx

```diff
@@ -18,19 +18,16 @@ export default function Demo() {
       }}
     >
       <Box
+        component="img"
         sx={{
           maxHeight: { xs: 233, md: 167 },
           maxWidth: { xs: 350, md: 250 },
         }}
-        clone
-      >
-        <img
-          alt="The house from the offer."
-          height="233"
-          width="350"
-          src="https://images.unsplash.com/photo-1512917774080-9991f1c4c750?auto=format&w=350&dpr=2"
-        />
-      </Box>
+        alt="The house from the offer."
+        height="233"
+        width="350"
+        src="https://images.unsplash.com/photo-1512917774080-9991f1c4c750?auto=format&w=350&dpr=2"
+      />
```

Comment:
That is not the same. `width` and `height` should end up on the actual `<img />` in the DOM. Now they're intercepted by the `Box`: https://deploy-preview-23686--material-ui.netlify.app/system/basics/#demo

This is a good example why the `component` prop is problematic. It makes it hard to reason about where props end up.

Reply:
In this case, this won't be a problem, but I see your point. I think that once we actually remove the support for all box props except the `sx` prop this will no longer be an issue.

Now that we have the codemod, I think we can move with testing it on the docs and if all occurrences are correctly replaced, we can just drop them. What do you think?

Reply:
For clarity, should we move the height and width to the `sx` styles?

Reply:
> For clarity, should we move the height and width to the `sx` styles?

Done